### ISSUE TITLE
lack of quarto templates for academic articles and theses

### ISSUE THREAD

**Issue by zawkzaw** [quarto-dev/quarto-cli#170]
It will benefit the academic community 
if you can elaborate on how to produce academic articles and theses using quarto book project like templates included in 'rticles', 'papaja' and 'thesisdown' . 

It is also very useful if we can adapt these templates in quarto because quarto is more userfriendky and versatile in choice of programming language used in data analysis.

**Comment by jjallaire** [comment]
Yes, we are planning on working on this very soon. The idea will be to create a framework for producing these sorts of templates as well as porting a bunch of the existing 'rticles' templates.

**Comment by iandol** [comment]
There are lua filters for academic metadata (critical for most academic output), available via: 

https://github.com/pandoc-scholar/pandoc-scholar and https://github.com/pandoc/lua-filters 

I'm not an rticles user

**Comment by jjallaire** [comment]
See also JAMS (discussed here: https://github.com/quarto-dev/quarto-cli/issues/287)

**Comment by ulyngs** [comment]
I'm super excited to see what template support in Quarto will look like!

In your work on this, I wonder if it's possible to automate some of the steps in making a LaTeX template work with Quarto in the first place.

In particular, I wonder if it'd be possible to automatically add the LaTeX calls into the template that's required to make this work:

1. code chunks with `echo=TRUE` show up in the output rather than throwing a `! LaTeX Error: Environment Shaded undefined.` error
2. inclusion of references with CSL, biblatex, or natbib

At the moment, I think many non-expert users of R Markdown gets stuck on using a LaTeX template, because the four steps involved in adapting a LaTeX template for this are not obvious (I wrote [this blog post](https://ulriklyngs.com/post/2021/12/02/how-to-adapt-any-latex-template-for-use-with-r-markdown-in-four-steps/#step-3-enable-code-inclusion-1) to document these steps).

**Comment by jjallaire** [comment]
Yes, we will definitely have a solution for all of this. There will be a system of template partials whereby you can override just parts of the default pandoc template (therefore preserving things like the highlighting macros). Some work in progress / examples are here: https://github.com/quarto-dev/quarto-cli/tree/main/tests/docs/templates (they'll be much more on the way of documentation for this soon).

**Comment by mingsu** [comment]
Is it possible to unify the YAML keywords across of Journal Templates.

So there is no need to update the whole YAML structure but change the `publisher` key.


```
---
title: "A journal article"
publisher: "elsevier" # or "acs"
...
---
```

**Comment by dragonstyle** [comment]
We are definitely seeking to accomplish this, starting with authors and their affiliations, but in general, it is definitely a goal to use the same document with different templates to get meaningful and appropriate output. Here's an example of where we're heading:

https://github.com/quarto-dev/quarto-cli/blob/main/tests/docs/extensions/format/academic/document.qmd

**Comment by andersgs** [comment]
Thank you for Quatro. I was just looking at the template referenced in the link above. Is country a possible variable for affiliation?
